resources/lang/id/contest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'header' => [
+        'small' => 'Karena sesungguhnya persaingan tidak hanya terjadi di dalam lapangan permainan.',
+        'large' => 'Kompetisi Komunitas osu!',
+    ],
+    'voting' => [
+        'over' => 'Pemungutan suara untuk kontes ini telah berakhir',
+        'login_required' => 'Silakan masuk untuk memilih.',
+        'best_of' => [
+            'none_played' => 'Anda tidak tercatat pernah memainkan map-map yang disyaratkan dalam kontes ini!',
```

Comment:
Tampaknya anda belum pernah memainkan map-map yang diakui dalam kontes ini!

Reply:
yang diakui?

Reply:
"any beatmaps that qualify for this contest"

Beatmap yang memenuhi syarat?

Reply:
Umm, I'm not entirely sure but I -think- this concerns the Community Choice contests (for example http://osu.ppy.sh/community/contests/59), in which in order for you to be able to cast your vote to a a specific map you'll have to play that map previously first

Reply:
In regards to that, umm... I'm not entirely sure but I think this would solve the problem here : 

+            'none_played' => 'Tampaknya Anda belum pernah memainkan map-map yang tercatat diikutsertakan dalam kontes ini!',